presto-product-tests/bin/product-tests-suite-6-non-generic.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+# Suite of product tests which are *not* run on ${HADOOP_BASE_IMAGE} and therefore
```

Comment:
can you verify if `${HADOOP_BASE_IMAGE}` is different than default, then we can either fail without running tests or ignore tests.